model-optimizer/unit_tests/utils/graph.py

```diff
@@ -276,25 +276,30 @@ def __getitem__(self, item):
                                           'shape': int64_array(shape) if shape is not None else None}}
 empty_data = lambda name: valued_data(name, None)
 
-shaped_parameter = lambda name, shape: {**regular_op(name, {'op': 'Parameter', 'shape': shape,
-                                                            'infer': Parameter.infer}),
-                                        **shaped_data(name + '_d', shape)}
+shaped_parameter = lambda name, shape, kwargs={}: {**regular_op(name, {'op': 'Parameter', 'type': 'Parameter',
+                                                                       'shape': shape, 'infer': Parameter.infer,
+                                                                       **kwargs}),
+                                                   **shaped_data(name + '_d', shape)}
 
-result = lambda name=None: {name if name is not None else 'output': {'kind': 'op', 'type': 'Result', 'op': 'Result',
-                                                                     'infer': lambda x: 0}}
+result = lambda name='output': {name: {'kind': 'op', 'type': 'Result', 'op': 'Result', 'infer': lambda x: 0}}
 
 regular_op_with_shaped_data = lambda name, shape, kwargs: {**regular_op(name, kwargs),
                                                            **shaped_data(name + '_d', shape)}
 regular_op_with_empty_data = lambda name, kwargs: {**regular_op(name, kwargs), **empty_data(name + '_d')}
 
 # constants
-const = lambda name, value: {name: {'kind': 'op', 'value': value, 'shape': int64_array(value.shape),
-                                    'type': 'Const', 'infer': Const.infer, 'op': 'Const'}}
-fake_const = lambda name, shape: {name: {'kind': 'op', 'value': None, 'infer': Const.infer,
-                                         'shape': int64_array(shape) if shape is not None else None}}
-shaped_const_with_data = lambda name, shape: {**fake_const(name, shape), **shaped_data(name + '_d', shape)}
+const = lambda name, value, kwargs={}: {name: {'kind': 'op', 'value': value, 'shape': int64_array(value.shape),
+                                               'type': 'Const', 'infer': Const.infer, 'op': 'Const', **kwargs}}
 
-valued_const_with_data = lambda name, value: {**const(name, value), **valued_data(name + '_d', value)}
+fake_const = lambda name, shape, kwargs={}: {name: {'kind': 'op', 'op': 'Const', 'type': 'Const',
+                                                    'value': None, 'infer': Const.infer, **kwargs,
```

Comment:
Let's add `type_infer` to the dictionary.

Reply:
done